third_party/move/move-prover/move-docgen/src/docgen.rs

```diff
@@ -1197,7 +1324,31 @@ impl<'env> Docgen<'env> {
             self.begin_collapsed(title);
         }
         for block in blocks {
-            self.doc_text(self.env.get_doc(&block.loc));
+            let text = self.env.get_doc(&block.loc);
+            let start_tag = "<spec-table>";
+            let end_tag = "</spec-table>";
+
+            if let Some(start) = text.find(start_tag) {
+                if let Some(end) = text.find(end_tag) {
+                    let table_text = text[start + start_tag.len()..end].trim();
+                    self.doc_text(&text[0..start]);
+                    self.section_header("High-level Specification", "high-level-spec");
```

Comment:
Did we always called this "Specification" @junkil-park @rahxephon89? I thought "High-Level Properties" or "Requirements"? Specification is a bit problematic because of the overload with formal spec (which this is not). Anyway, we can update this later.

Reply:
I think we use the term `high-level requirements` and we should be consistent between here and the blog post. cc @aalok-t @junkil-park

Reply:
I have uniformly used requirements in the update.